clang/tools/dpct/DpctOptRules/python_setup_script_migration_rule.yaml

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) Intel Corporation
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# See https://llvm.org/LICENSE.txt for license information.
+
+# This file will be installed to folder:
+# {dpct_install_folder}/extensions/opt_rules.
+# The rule specified in this file can be imported with commandline option:
+# --rule-file={dpct_install_folder}/extensions/opt_rules/setup_rules/setup_script_migration_rule.yaml
+
+- Rule: rule_import_torch
+  Kind: PySetupRule
+  Priority: Fallback
+  MatchMode: Partial
+  PythonSyntax: import_torch
+  In: import torch
+  Out: |
+    import torch
+    import intel_extension_for_pytorch
+    SYCL_HOME = True
+
+- Rule: rule_from_torch_utils_import_cpp_extension_only
+  Kind: PySetupRule
+  Priority: Fallback
+  MatchMode: Partial
+  PythonSyntax: from_torch_utils_import_cpp_extension_only
+  In: from torch.utils import cpp_extension
+  Out: from intel_extension_for_pytorch.xpu import cpp_extension
+
+- Rule: rule_torch_utils_cpp_extension
+  Kind: PySetupRule
+  Priority: Fallback
+  MatchMode: Partial
+  PythonSyntax: torch_utils_cpp_extension
+  In: torch.utils.cpp_extension
+  Out: intel_extension_for_pytorch.xpu.cpp_extension
```

Comment:
what if the pytorch is stock pytroch not IPEX?  
how about prepare 2 copy of the file, one for stock pytoch,  another for IPEX?  
chnage python_setup_script_migration_rule.yaml to python_setup_script_migration_rule_pytorch.yaml  and python_setup_script_migration_rule_ipex.yaml